cluster/images/etcd/migrate/versions.go

```diff
@@ -187,12 +177,3 @@ func ParseSupportedVersions(s string) (SupportedVersions, error) {
 	}
 	return versions, nil
 }
-
-// MustParseSupportedVersions parses a comma separated list of etcd versions or panics if the parse fails.
-func MustParseSupportedVersions(s string) SupportedVersions {
-	versions, err := ParseSupportedVersions(s)
```

Comment:
This one too:
https://github.com/kubernetes/kubernetes/blob/c7f9dd0bafe2f3328148a85eed9c18322b9f308e/cluster/images/etcd/migrate/integration_test.go#L43

Reply:
The integration_test.go is unused， please confirm.

Reply:
@hwdef what do you mean `The integration_test.go is unused`?

Reply:
@RainbowMango 
![image](https://user-images.githubusercontent.com/13084946/63670226-b7c8d380-c80e-11e9-820d-67e5c71ba682.png)

Reply:
@wenjiaswe 
I think I better not change it,because i don not know why the test file not in the list.
I will close this RP.
Maybe someone will change it later.
Thanks for your suggestion.